Formula/ssm.rb

```diff
@@ -1,9 +1,9 @@
 class Ssm < Formula
   desc "ssh replacement: CLI program that wraps SSM's EC2 Run Command"
   homepage "https://github.com/guardian/ssm-scala"
-  version "2.3.0"
-  url "https://github.com/guardian/ssm-scala/releases/download/v2.3.0/ssm.tar.gz"
-  sha256 "dfef888b9a1d21da4137cee1b73431ac538379856bcf8663a3c02cad3df1ea1d"
+  version "2.4.0"
+  url "https://github.com/guardian/ssm-scala/releases/download/v2.4.0/ssm.tar.gz"
+  sha256 "66e1d75d7eafbcab4eb68909c7415c7aa7bc475d2e6aed9c1122744d56fed468"
```

Comment:
This value appears to be incorrect.

```sh
curl -sL https://github.com/guardian/ssm-scala/releases/download/v2.4.0/ssm.tar.gz | shasum -a 256
```

Gives the value as `d16deabe57596adbc242fe953b24d2962aa81178ee9ab9ddeb701b6581c6da22`.

Reply:
Oh thank you @akash1810 . I was thinking checking but decided OP could not have possibly mis-computed it. Thanks for being thorough :)

Reply:
Thanks both – I'd added the sha of the raw binary, not thinking that homebrew would be using the tar! 🤦

Reply:
No worries Jonathon, I actually could have documented that better. It's actually hidden here in this file: https://github.com/guardian/ssm-scala/blob/358c418edc553381eb64b442eb1c652c25f12f3a/generate-executable.sh#L13